library/me_calculator.inc.php

```diff
@@ -0,0 +1,107 @@
+<?php
+/*
+* Morphine Calculator
+* @package OpenEMR
+* @link http://www.open-emr.org
+ * @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @copyright Copyright (c) 2016-2017 Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+*/
+
+
+
+      /*
+      *  This section is to save the ME total 
+      *
+      *
+      */
+    function saveMEinfo($pid, $enc, $sum)
+    {
+        if(empty($enc)){
+            return "<font color='red'>Please select an encounter to save this data</font>";
+        } else {
+        sqlStatement("INSERT INTO `morphine_eq` (`id`, `datetime`, `encounter`, `last`, `next`, `notes`, `pid`)". 
+                      "VALUES ('', NOW(), $enc, $sum, '', '', $pid)");
+                      
+                      return "Saved";
+        }
+    }
+	/*
+	*
+	*  Retrieve last ME recorded
+	*
+	*/
+	
+    function fetchMEs($pid)
+    {
+         $sql = "SELECT * FROM morphine_eq WHERE pid = ? ORDER BY id DESC LIMIT 1";
+         $query = sqlStatement($sql,$pid);
```

Comment:
This line should not work. need this instead:
``````
$query = sqlStatement($sql, array($pid));
``````

Reply:
It does work but I changed it to the standard notation.